types/react-native-web/README.md

```diff
@@ -0,0 +1,17 @@
+# `react-native-web`
```

Comment:
Pls remove this file as this is not published as part of DT types and is only useful for type writers benefit than users benefit (eg you want to generate types using some script, instructions for those scripts can go in here)

Reply:
@sheetalkamat Done👌